docs/examples/howto/howto.md

```diff
@@ -13,7 +13,7 @@ to run example workflows (see [](python-bindings)).
 
 :::{toctree}
 :maxdepth: 1
-
+add_new_algorithm
```

Comment:
Actually, this is not showing up in the contens-list of the readthedocs. I think its added to the wrong toctree (the 
`docs/examples/howto/howto.md`), but that is not visible in the readthedocs. I think the correct one would be `docs/howto/howto.md`

Reply:
I have changed this. Unfortunately Have lost env to recompile the docs locally. (missing myst parser).

Reply:
Updated. After a successful restore of env I confirm that I see the doc. 
Anyways, having Howto twice docs tree is a bit confusing.

Reply:
I think that's done.